suite/tests/CMakeLists.txt

```diff
@@ -1308,9 +1308,13 @@ function(torun test key source native standalone_dr dr_ops exe_ops added_out)
       # We always run infloop.
       get_target_path_for_execution(app_path linux.infloop "${location_suffix}")
     else ()
-      # It's easier to run our own infloop, which prints so we know when it's up,
-      # than to run notepad or sthg.
-      get_target_path_for_execution(app_path win32.infloop "${location_suffix}")
+      if ("${runall}" MATCHES "<attach>")
+        get_target_path_for_execution(app_path win32.attachee "${location_suffix}")
```

Comment:
Add a comment explaining why attach isn't using infloop since it seems like infloop would be perfect for it

Reply:
Done